### DISCUSSION TITLE
is there any details on how to use the load balancer or proxy to terminate TLS/HTTPS

### DISCUSSION THREAD

**Discussion by njalan** [trinodb/trino#15928] [Q&A]
I want to connect the trino which enables SSL without certificate files. Is there any details on below approach:
Use the [load balancer or proxy](https://trino.io/docs/current/security/tls.html#https-load-balancer) at your site or cloud environment to terminate TLS/HTTPS. This approach is the simplest and strongly preferred solution

**Comment by luhea** [comment]
Hello，njalan，I have same question，can you resolved it？

**Comment by njalan** [comment]
Put the related certificated on F5 server then client  can connect to F5 server without certificate.

**Reply by luhea** [reply]
ok，thanks, i did not have a F5, I hava a haproxy,I did not config it. Who can shared config load balancer in Trino?
